### ISSUE TITLE
Multipart/form-data request with compressed params

### ISSUE THREAD

**Issue by ArVan** [balderdashy/skipper#64]
Hi there. I am implementing an api which works with mobile apps. Now there is a POST request in my backend which receives some images and text parameters. The request is `multipart/form-data`.
We are now trying to implement request compression. The mobile app constructs request where each text parameter and attached file are being `gzipped`. So when the request reaches my controller, I see the value of my text param as something like this:
```
'\u001f�\b\u0000\u0000\u0000\u0000\u0000\u0002\u0003͑]o�0\u0014��˹&�-�\u0000��]/��n\u0016�\u00148�F��T�1��\u0015��,3q��w��m�s���-\u001cb�B���|c�R�]���"�M�:t��5d\u0010C\u0004�\f\u001f�`2\u0017,�\u0015qBҜa�r��\'\u0012͸`�ﵪq���xQgp�\n[{g�\n]��=$/?A\u0004�\u001d�J�H��Yc�񺁌0\u001a\u000b�yB\b���\f�^��M"(�3\u0005��-�\n�����(#\n�y\u0004؅����\u0012�zg�E��@ӯ\u001eFU��U8\u000e˝z~�\u0000>\f`o0Kc\u001eAH\u00077�h\u000fX\u0007��\u001c�r]�\u0016�n���辗k���T\u0015�����&���<Q�\u000b\u001ei������W��o�[�\u0004��S���F\u0005I9��(���@!i�\bF�iF%�,�B�b4��Q~���"��\u0011���FӄHy6J�A�\'Sq.�`g*r\u000f*1�J�������\u000b➿��\u0006\u0000\u0000'
```

I'm not sure what this is and how to decode this. Is Skipper handling compressed multipart requests?
I know that other type of requests like `application/json` get to my controller as normal `json` body. So the thing is with `multipart` content type.

**Comment by sailsbot** [comment]
Thanks for posting, @ArVan.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
